src/review-list/review-list.component.tsx

```diff
@@ -67,18 +67,23 @@ const ApproveTestMenu: React.FC<ApproveResultMenuProps> = ({
 const ReviewList: React.FC<ReviewlistProps> = ({ fulfillerStatus }) => {
   const { t } = useTranslation();
 
-  const [activatedOnOrAfterDate, setActivatedOnOrAfterDate] = useState("");
-
   const { workListEntries, isLoading } = useGetOrdersWorklist(fulfillerStatus);
 
+  const filtered = workListEntries?.filter(
```

Comment:
Let's rename this as well to match the records `filteredOrders` or something else could be better